packages/cli/src/commands/access/update.ts

```diff
@@ -0,0 +1,42 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+import {isTeamApp} from '../../lib/access/utils'
+
+export default class Update extends Command {
+    static topic = 'access';
+    static description = 'update existing collaborators on an team app';
+    static flags = {
+      permissions: flags.string({
+        char: 'p',
+        description: 'comma-delimited list of permissions to update (deploy,manage,operate)',
+        aliases: ['privileges'],
+        deprecateAliases: true,
```

Comment:
Shouldn't we remove this already? I mean, given that it was already deprecated in 8.x and showed a deprecation message. Maybe moving to 9.x we shouldn't accept the alias anymore.

Reply:
agreed, removed